src/services/sightings.ts

```diff
@@ -1,15 +1,26 @@
 import { Sighting } from '../models/sighting';
-import { User } from 'discord.js';
+import { Guild, User } from 'discord.js';
 import { DateTime } from '../util/date-time';
 
 export class Sightings {
-    public static async list(license: string, limit = 10): Promise<string | null> {
+    public static async list(license: string, discordGuildId: string | null, limit = 10): Promise<string | null> {
+        let where = {};
```

Comment:
```
let where = {
      license: license, 
};

if (discordGuildId != null) {
    where.discordGuildId = discordGuildId=
}

Reply:
![image](https://user-images.githubusercontent.com/9088316/203623933-d914f3a4-3808-4865-96e2-7692b8faba73.png)

kan niet, `discordGuildId` is geen property op `where`